### PR TITLE
Add --version (-v) argument to CLI

### PR DESCRIPTION
**Pull request by jeremylinlin**
- This commit fixes https://github.com/operator-framework/operator-courier/issues/24

### CODE REVIEW

File: operatorcourier/cli.py [hunk 1]
```diff
@@ -16,7 +17,12 @@ class _CliParser():
     """
 
     def __init__(self):
-        pass
+        # Several different ways of maintaining a single version number for the project
+        # https://packaging.python.org/guides/single-sourcing-package-version/
```
Comment (awgreene) [inline operatorcourier/cli.py]: Do we really need to include this comment? We've chosen the method to manage the version.

File: operatorcourier/cli.py [hunk 2]
```diff
@@ -59,7 +65,7 @@ def push(self):
         args, leftovers = parser.parse_known_args(sys.argv[2:])
         api.build_verify_and_push(args.namespace, args.repository, args.release, args.token, source_dir=args.source_dir)
 
-    # Parse the push command
+    # Parse the nest command
```
Comment (awgreene) [inline operatorcourier/cli.py]: Please make this change in a separate commit.

File: operatorcourier/cli.py [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import argparse
 import sys
 from operatorcourier import api
+import re
```
Comment (kevinrizza) [inline operatorcourier/cli.py]: can we order imports like:

1. stdlib
2. third party
3. package dependencies

File: operatorcourier/cli.py [hunk 4]
```diff
@@ -16,7 +17,10 @@ class _CliParser():
     """
 
     def __init__(self):
-        pass
+        with open('setup.py', 'r') as f:
```
Comment (kevinrizza) [inline operatorcourier/cli.py]: shouldn't we only do this in the case that the -v command is specified? rather than during the init

File: operatorcourier/cli.py [hunk 5]
```diff
@@ -19,6 +21,11 @@ def __init__(self):
         pass
 
     def parse(self):
+        def version():
+            with open('setup.py', 'r') as f:
```
Comment (MartinBasti) [inline operatorcourier/cli.py]: This won't work, `setup.py` is not a part of installed package, this works only from repository locally.

```bash
$ virtualenv venv-3
$ source venv-3/bin/activate
$ pip install git+https://github.com/jeremylinlin/operator-courier.git@add-version-arg-to-cli
$ operator-courier -v                                                                                                    
[Errno 2] No such file or directory: 'setup.py'
```

Comment (MartinBasti) [reply operatorcourier/cli.py]: Please consult this: https://packaging.python.org/guides/single-sourcing-package-version/

File: operatorcourier/cli.py [hunk 6]
```diff
@@ -1,7 +1,10 @@
 import argparse
+import pkg_resources
 import sys
+import re
```
Comment (MartinBasti) [inline operatorcourier/cli.py]: unused import

**Comment by jeremylinlin** [comment]
> Thank you for addressing my comments!

Thank you for your very useful comments as well!
